R/merge_sce_list.R

```diff
@@ -156,12 +166,71 @@ merge_sce_list <- function(sce_list = list(),
     metadata_list$sample_metadata <- sample_metadata
   }
 
-  # Create the merged SCE from the processed list ------------------
+
+  ## Handle altExps ------------------------------------------------------
+
+  # If we are including altExps, process them and save to list to add to merged SCE
+  merged_altexps <- list()
+  if (include_altexp) {
+
+    # First we need to determine the final column names of the merged_sce (not yet made)
+    #  for use in altExp code. Later we'll apply this order to the merged_sce itself.
+    merged_colnames <- sce_list |>
+      purrr::map(colnames) |>
+      unlist() |>
+      unname()
```

Comment:
```suggestion
    all_merged_barcodes <- sce_list |>
      purrr::map(colnames) |>
      unlist() |>
      unname()
```
This really confused me when reviewing this. I thought you were grabbing the `colnames` to use to fill in the `colData`. Can we make this variable name clear that you are grabbing all the barcodes? I know that the columns have the batch ID and barcode, so maybe we just make that clear that's what your grabbing in the comment. Maybe also `all_merged_colnames` instead of `all_merged_barcodes`? This also could just be a me problem.

Reply:
> This also could just be a me problem.

Your confusion makes sense, will take suggestions!